hack/.quickstartEnv

```diff
@@ -0,0 +1,44 @@
+# shellcheck shell=bash
```

Comment:
I see the reason for moving this out to a separate file.
However, I wonder if it's necessary, given the comment below.